src/Native/Unix/System.Native/pal_io.c

```diff
@@ -385,7 +385,7 @@ int32_t SystemNative_ReadDirR(DIR* dir, void* buffer, int32_t bufferSize, struct
     // 0 returned with null result -> end-of-stream
     if (result == NULL)
     {
-        *outputEntry = {}; // managed out param must be initialized
+        memset(outputEntry, 0, sizeof(*outputEntry)); // managed out param must be initialized
```

Comment:
Is this so that it zeroes padding as well? Is that something this API requires?

Reply:
This change snuck in to make the native parts of the code compile on this machine. It is unrelated to the StartTime functionality. It is a missed change when moving the native parts from cpp to c. I can put it in a separate PR if you want.

Reply:
> I can put it in a separate PR if you want.

Thanks for fixing.  Please separate it out.

Reply:
Created https://github.com/dotnet/corefx/pull/25365